### ISSUE TITLE
automatic startup doesn't seemt to work

### ISSUE THREAD

**Issue by davidermined** [digitalcreations/MaxTo#347]
Restarted my computer and automatic startup was enabled by default, but no MaxTo in the notification area.

**Comment by vegardlarsen** [comment]
Did MaxTo start at all? Can you check [the logs](https://maxto.net/documentation/troubleshooting) for any issues relating to startup tasks?

**Comment by khongi** [comment]
I have the same issue. Checked the logs after restarting, MaxTo didn't seem to start, but there was an error just before shutting down.

```
2019-02-27 08:10:51 [Core@2.0.0.456] [Information] Found startup task but MaxTo is not currently set to start with Windows. Removing task.
2019-02-27 08:10:51 [Core@2.0.0.456] [Error] Could not register startup task after setting changed.
System.UnauthorizedAccessException: Access is denied. (Exception from HRESULT: 0x80070005 (E_ACCESSDENIED))
   at Microsoft.Win32.TaskScheduler.V2Interop.ITaskFolder.DeleteTask(String Name, Int32 flags)
   at Microsoft.Win32.TaskScheduler.TaskFolder.DeleteTask(String name, Boolean exceptionOnNotExists)
   at MaxTo.Core.Settings.LoginRegistrationService.ConfigurationChanged(Object sender, EventArgs args)
```
Here is the full log:
[maxto20190227.log](https://github.com/digitalcreations/MaxTo/files/2908966/maxto20190227.log)

**Comment by vegardlarsen** [comment]
Copied from my response at #349:

My guess is that a previous version of the startup task is present (from the previous version of MaxTo), and may need to be removed manually. If that is the case; open Task Scheduler, and manually delete the task called "MaxTo" (maybe in lowercase).

![image](https://user-images.githubusercontent.com/436508/53549506-9bd0c780-3b34-11e9-89d4-eba63eb7c6a7.png)

**Comment by khongi** [comment]
I removed the old task ![maxto_old_task](https://user-images.githubusercontent.com/11983684/53552432-9d51be00-3b3b-11e9-9561-bfd23d91f7b3.jpg)

Seemed like something wrong with the path. Then I started MaxTo, it recreated the task, but after restart still no MaxTo. Same problem.
![maxto_new_task](https://user-images.githubusercontent.com/11983684/53552884-ad1dd200-3b3c-11e9-96ae-1c8a1870af74.PNG)

I checked the error code and looks like if you remove the `file:\` part from Action\Start in (optional) 
![maxto_task_startin](https://user-images.githubusercontent.com/11983684/53553216-6a102e80-3b3d-11e9-9bb4-53171f3948e8.PNG)

Then the task runs successfully.

![maxto_task_successful](https://user-images.githubusercontent.com/11983684/53553222-6f6d7900-3b3d-11e9-8404-4306aa7d19c6.PNG)

**Comment by samusaran** [comment]
Can confirm @khongi solution. Removing `file:\` or the entire `Start in` path makes the task run succesfully.

**Comment by vegardlarsen** [comment]
Thank you for providing this feedback. This should make it easier to fix this issue.

**Comment by vegardlarsen** [comment]
I have fixed this issue, and the changes will be in 2.0.0-rc.2.

**Comment by MalteHillmann** [comment]
@vegardlarsen I have an issue maybe related to this:
MaxTo-TaskScheduler seems to work. In Task-Manager MaxTo.Core.exe is running after logon. The other tasks are missing. If I try to start MaxTo manually, it says "MaxTo is already running, you should find it in the notification area."
I have to kill MaxTo.Core.exe in taskmgr and start MaxTo manually again to get it working.

_(by the way: why is this error message not translated?)_
